src/murmur/Messages.cpp

```diff
@@ -77,18 +77,20 @@
 		sendMessage(uSource, mppd); \
 	}
 
-/// A helper class for managing temporary access tokens
+/// A helper class for managing temporary access tokens.
 /// It will add the tokens in the comstructor and remove them again in the destructor effectively
-/// turning the tokens into a scope-based property
+/// turning the tokens into a scope-based property.
 class TemporaryAccessTokenHelper {
 	protected:
 		ServerUser *affectedUser;
 		QStringList qslTemporaryTokens;	
 		Server *server;
 
 	public:
-		TemporaryAccessTokenHelper(ServerUser *affectedUser, const QStringList &tokens, Server *server) : affectedUser(affectedUser),
-			qslTemporaryTokens(tokens), server(server) {
+		TemporaryAccessTokenHelper(ServerUser *affectedUser, const QStringList &tokens, Server *server)
+			: affectedUser(affectedUser)
+			 , qslTemporaryTokens(tokens)
+			 , server(server) {
```

Comment:
This should be indented using spaces.

We should probably set up a code style checker on one of the CI services, it's very easy to miss whitespace inconsistencies.